src/Timezone/TimezoneAwareTrait.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\IntlBundle\Timezone;
+
+/**
+ * Basic Implementation of TimezoneAwareInterface.
+ *
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+trait TimezoneAwareTrait
+{
+    /**
+     * @var string|null
+     */
+    protected $timezone;
```

Comment:
```suggestion
    private $timezone;
```
Since there is an accessor…

Reply:
And how would you modify this?

Reply:
I change it to be private.
>And how would you modify this?

The trait expose their properties as part of the class that uses the trait, in a transparent way, like if they belong to that class.
The property can be safely modified by the methods of the class.

Reply:
TIL: You can override private fields oO

https://3v4l.org/e2G7D